File: Backend/Models/Ingredient.cs
using System;
using System.Collections.Generic;

namespace Backend.Models
{
    public partial class Ingredient
    {
        public Ingredient()
        {
            IngredientList = new HashSet<IngredientList>();
            Recipe = new HashSet<Recipe>();
        }

        public int Idingredient { get; set; }
        public string Name { get; set; }

        public virtual ICollection<IngredientList> IngredientList { get; set; }
        public virtual ICollection<Recipe> Recipe { get; set; }
    }
}

File: frontend/src/app/models/Ingredient.ts
export class Ingredient {
  idIngredientList: any;
  idIngredient: any;
  name: string;
  amount: string;
  constructor(idIngredientList: any, ingredient: string, amount: string, idIngredient: any) {
    this.idIngredientList = idIngredientList;
    this.name = ingredient;
    this.amount = amount;
    this.idIngredient = idIngredient;
  }
}
